apm-agent/src/test/resources/configuration.asciidoc.ftl

```diff
@@ -117,9 +117,15 @@ ${option.description}
 <#if option.dynamic><<configuration-dynamic, image:./images/dynamic-config.svg[] >></#if>
 
 <#if option.valueType?matches("TimeDuration")>
+  <#if option.valueConverter.canUseMicros>
+Supports the duration suffixes `us`, `ms`, `s` and `m`.
+  <#else>
 Supports the duration suffixes `ms`, `s` and `m`.
+  </#if>
 Example: `${option.defaultValueAsString}`.
+  <#if option.valueConverter.defaultDurationSuffix?has_content>
 The default unit for this option is `${option.valueConverter.defaultDurationSuffix}`.
+  </#if>
```

Comment:
To discourage relying on default durations, I think we should remove default durations from the docs.
```suggestion
```